cypress/integration/InstancesTest.ts

```diff
@@ -0,0 +1,140 @@
+        "Instance one"
+      );
+      cy.ouiaId("actions", "PF4/Dropdown")
+        .should("be.visible")
+        .within(() => {
+          cy.ouiaId("actions", "PF4/DropdownToggle").click();
+          cy.ouiaId("details", "PF4/DropdownItem").should("be.visible");
+          cy.ouiaId("delete", "PF4/DropdownItem").should("be.visible");
+        });
+    });
+
+    it("Details", () => {
+      cy.ouiaId("actions", "PF4/Dropdown")
+        .should("be.visible")
+        .within(() => {
+          cy.ouiaId("actions", "PF4/DropdownToggle").click();
+          cy.ouiaId("details", "PF4/DropdownItem").click();
+        });
+      //The InstanceDetails.tsx contains dummy data which are not related to this instance
+      cy.ouiaId("instance-details-panel")
+        .within(() => {
+          cy.ouiaId("instance-details-name", "PF4/Text")
+            .should("have.text", "Instance one")
+            .should("be.visible");
+          cy.ouiaId("instance-details-id")
+            .should("have.text", "3543edaa-1851-4ad7-96be-ebde7d20d717")
+            .should("be.visible");
+          cy.ouiaId("instance-details-owner")
+            .should("have.text", "bebianco@redhat.com")
+            .should("be.visible");
+          cy.ouiaId("instance-details-endpoint")
+            .find("input")
+            .should(
+              "have.value",
+              "https://ob-3543edaa-1851-4ad7-96be-ebde7d20d717.apps.openbridge-dev.fdvn.p1.openshiftapps.com/events"
+            )
+            .should("be.visible");
+          cy.ouiaId("instance-details-submitted-date")
+            .should("have.text", "Tuesday, April 12th, 2022 12:04 PM")
+            .should("be.visible");
+          cy.ouiaId("instance-details-published-date")
+            .should("have.text", "Tuesday, April 12th, 2022 12:06 PM")
+            .should("be.visible");
+          cy.ouiaId("close-instance-details").click();
+        })
+        //constructions like "not.exists" or "not.be.visible" fail in this case
+        .should("have.attr", "hidden");
+    });
+
+    it("Processors Content", () => {
+      cy.ouiaId("instance-details", "PF4/Tabs")
+        .ouiaId("processors", "PF4/TabButton")
+        .click();
+      cy.ouiaId("processors", "PF4/TabContent").within(() => {
+        cy.ouiaId("rows-toolbar", "PF4/Toolbar")
+          .ouiaId("create-processor", "PF4/Button")
+          .should("be.visible");
+        cy.ouiaId("Processors list table", "PF4/Table")
+          .ouiaId("f8f34af4-caed-11ec-9d64-0242ac120002", "PF4/TableRow")
+          .find("td")
+          .then(($cells) => {
+            expect($cells).have.length(6);
+            expect($cells.eq(0)).have.text("Processor three");
+            expect($cells.eq(1)).have.text(
+              "f8f34af4-caed-11ec-9d64-0242ac120002"
+            );
+            expect($cells.eq(2)).have.text("Source");
+            expect($cells.eq(3)).have.text("accepted");
+            expect($cells.eq(4)).have.text("27 days ago");
```

Comment:
I think this will change each day we start the test, can we check just substring? "days ago" for example?

Reply:
I agree with @jomarko that probably this is going to change every day!

Reply:
@jomarko @vpellegrino  My fault, thank you. It should be deleted.

Reply:
@sjamboro then it's fine from my side :ok_hand: